sklearn/model_selection/_split.py

```diff
@@ -1776,17 +1765,6 @@ def _validate_shuffle_split_init(test_size, train_size):
     NOTE This does not take into account the number of samples which is known
```

Comment:
This function is completely redundant with `_validate_shuffle_split` except that the former one is used in the `__init__` and the later one is used in the `split` function.

As we usually don't do validation in the `__init__` and the validation is done twice, do you mind if I remove the `__init__` validation ?